content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -290,12 +289,12 @@ Below you can find installation instructions for some popular Pod network plugin
 {{< tabs name="tabs-pod-install" >}}
 
 {{% tab name="Calico" %}}
-For more information about using Calico, see [Quickstart for Calico on Kubernetes](https://docs.projectcalico.org/latest/getting-started/kubernetes/), [Installing Calico for policy and networking](https://docs.projectcalico.org/latest/getting-started/kubernetes/installation/calico), and other related resources.
+[Calico](https://docs.projectcalico.org/latest/introduction/) is a networking and network policy provider. Calico supports a flexible set of networking options designed to give you the most efficient networking across a range of situations, including non-overlay and overlay networks, with or without BGP. Calico uses the same engine to enforce network policy for hosts, pods, and (if using Istio & Envoy) applications at the service mesh layer.
```

Comment:
@caseydavenport recently we had a purge of CNI providers in the kubeadm docs. SIG Docs have a KEP to remove third party documentation that attempts to advertise their product.

to comply with this can we simplify the above sentences to something more technical with less adjectives, as if it was written by a third party - possibly a single sentence listing the key features of latest 3.11.

> [Calico](https://docs.projectcalico.org/latest/introduction/) is a networking and network policy provider that supports .... x, y, z.

Reply:
@neolit123 Can you link to the KEP? This may help folks judge what is the appropriate level of content here. I do believe the goal should be to provide the level of content that is sufficient for a k8s user to judge whether an external project is of interest to them.  It shouldn't be marketing material but I think it does need to say what the project does at a level where an average reader of the docs would know if it is of interest in them learning more about.

Reply:
@lxpollitt the KEP is here:
https://github.com/kubernetes/enhancements/pull/1327